template/.gitignore

```diff
@@ -0,0 +1,4 @@
+/release/
+/{{ Executable }}
+
+coverage.out
```

Comment:
I feel like the `.drone.yml` should also be added here

Reply:
Fixed via https://github.com/drone-plugins/boilr-plugin/pull/6/commits/b7fe98ac474559ceaccb09dfb732e950f6d32649